system/Database/SQLSRV/Forge.php

```diff
@@ -233,46 +233,6 @@ protected function _processColumn(array $field): string
                 . $field['unique'];
     }
 
-    /**
-     * Process foreign keys
-     *
-     * @param string $table Table name
-     *
-     * @return string
-     */
-    protected function _processForeignKeys(string $table): string
```

Comment:
You cannot simply delete a protected method as userland code might be depending on this. If this is no longer needed, then deprecate the method.